aqt/helper.py

```diff
@@ -315,6 +334,10 @@ def response_timeout(self):
     def max_retries(self):
         return self.config.getfloat("requests", "max_retries", fallback=5)
 
+    @property
+    def max_retries_on_checksum_error(self):
+        return self.config.getint("requests", "max_retries_on_checksum_error", fallback=int(self.max_retries))
```

Comment:
This adds an extra setting to `settings.ini` that allows you to configure how many times `aqt` will retry a download when a checksum error occurs. I think that it's useful to differentiate between this and `max_retries`, which determines how many times a connection should be retried; these are different concepts and should be handled differently.

I worry that it will be confusing to a user to have these two settings; the difference is unclear unless you read the code. Maybe `max retries` could be renamed.

I don't know if falling back to `max_retries` is wise, or even makes sense.

Reply:
We can mark `max_retries` as deprecated and introduce new parameter `max_retries_on_connection_error` that fallback to `max_retries`. Does it help users?

Reply:
> We can mark `max_retries` as deprecated and introduce new parameter `max_retries_on_connection_error` that fallback to `max_retries`. Does it help users?

That sounds like a good idea. So here's how `Settings` would work:
1. Try to load `max_retries`, and fallback to 5.
2. Try to load `max_retries_on_connection_error`, and fallback to `max_retries`.
3. Try to load `max_retries_on_checksum_error`, and fallback to `max_retries`.

I'm not sure how to mark `max_retries` as deprecated; would we do that in the documentation somewhere?